reversion_compare/compare.py

```diff
@@ -294,45 +227,29 @@ def changed(self):
 
         return self.compare_obj1 != self.compare_obj2
 
-    def _get_result(self, compare_obj, func_name):
-        func = getattr(compare_obj, func_name)
-        result = func()
-        return result
-
-    def _get_both_results(self, func_name):
-        result1 = self._get_result(self.compare_obj1, func_name)
-        result2 = self._get_result(self.compare_obj2, func_name)
-        return (result1, result2)
-
     def to_string(self):
-        return self._get_both_results("to_string")
+        return self.compare_obj1.to_string(), self.compare_obj2.to_string()
 
     def get_related(self):
-        return self._get_both_results("get_related")
+        return self.compare_obj1.get_related(), self.compare_obj2.get_related()
 
     def get_many_to_many(self):
-        # return self._get_both_results("get_many_to_many")
-        m2m_data1, m2m_data2 = self._get_both_results("get_many_to_many")
-        return m2m_data1, m2m_data2
+        return self.compare_obj1.get_many_to_many(), self.compare_obj2.get_many_to_many()
 
     def get_reverse_foreign_key(self):
-        return self._get_both_results("get_reverse_foreign_key")
-
-    M2O_CHANGE_INFO = None
+        return self.compare_obj1.get_reverse_foreign_key(), self.compare_obj2.get_reverse_foreign_key()
 
     def get_m2o_change_info(self):
-        if self.M2O_CHANGE_INFO is not None:
```

Comment:
Since None is falsey, it seemed redundant.  I can restore if you prefer being explicit.